pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -44,6 +44,14 @@ import (
 	token "k8s.io/kubernetes/pkg/serviceaccount"
 )
 
+const (
+	// The annotation key used in the audit event that is persisted to the '/token' endpoint for service accounts.
+	// This annotation indicates the generated credential identifier for the service account token being issued.
+	// This is useful when tracing back the origin of tokens that have gone on to make request that have persisted
+	// their credential-identifier into the audit log via the user's extra info stored on subsequent audit events.
+	credentialIDAuditAnnotationKey = "authentication.kubernetes.io/issued-credential-id"
```

Comment:
docs lgtm

if we're going to use it publicly, putting this alongside the existing constant in the other package seems helpful

Reply:
done